Больше деталей

tymanesss opened the pull request:
---
:mortar_board: [Больше деталей](https://up.htmlacademy.ru/javascript/26/user/2067603/tasks/9)

:boom: https://htmlacademy-javascript.github.io/2067603-keksobooking-26/4/

tymanesss committed:
Update Readme.md

tymanesss committed:
Update Readme.md

tymanesss committed:
Merge branch 'master' of https://github.com/tymanesss/keksobooking-project into module1-task1

tymanesss committed:
Merge branch 'master' of github.com:htmlacademy-javascript/2067603-keksobooking-26

tymanesss committed:
Merge branch 'master' of github.com:htmlacademy-javascript/2067603-keksobooking-26

tymanesss committed:
More details

tymanesss committed:
More details (+ESlint corrections)

tymanesss committed:
More details (+ESlint corrections)

tymanesss committed:
More details (+ ESlint corrections)

copperkraft commented on js/main.js:
```diff
@@ -11,3 +11,108 @@ function getRandomFloat() {
+
+const getRandomPictureLink = () => {
+  const result = getRandomInt(1, 10);
+  return result < 10 ? `0${result}` : result;
```
Для добавления 0 лучше воспользоваться методом строки `.padStart()`
Также, у тебя фунция может вернуть как строку, так и число. Лучше избегать таких ситуаций, т.к. это может привести к непредсказуемым последствиям.

copperkraft commented on js/main.js:
```diff
@@ -11,3 +11,108 @@ function getRandomFloat() {
+};
+
+function getRandomArray(elements) {
+  const Array = elements.slice(getRandomInt(0, elements.length - 1));
```
`Array` это зарезервированое слово JS, его нельзя использовать для имени переменной.
Также, лучше называть массивы единообразно существительными во множественном числе. Чаще всего название может быть как-то связанно с бизнес-логикой, для "абстрактных" случаев лучше воспользоваться словом _items_, (для элементов HTML - _elements_)

copperkraft commented on js/main.js:
```diff
@@ -11,3 +11,108 @@ function getRandomFloat() {
+);
+
+const createAd = () => {
+  const getLocationLat = getRandomFloat(35.65000, 35.70000);
```
Значения для случайной генерации, такие как верхний/нижний пределы и т.д. лучше заносить в константы.

copperkraft commented on js/main.js:
```diff
@@ -11,3 +11,108 @@ function getRandomFloat() {
+  'hotel'
+];
+
+const CHECKIN = [
```
Чтобы очевиднее обозначить тот факт, что тут хранится массив, лучше использовать существительное во множественном числе, например _CHECKIN_TIMES_

tymanesss committed:
Corrections made

1. Изменила все функции на стрелочные;
2. Видоизменила первые две функции со случайными числами;
3. Добавила константы;
4. Добавлен метод .padStart()
5. Изменено имя одной из переменных (с Array на items)

tymanesss committed:
ESlint corrections

tymanesss committed:
ESlint corrections

tymanesss committed:
ESlint corrections
